app/services/api.ts

```
@@ -0,0 +1,157 @@
+  resBody: ApiResponseBody<T>,
+  {
+    skipAssertData = false,
+  }: {
+    /** Some special APIs return 204 No Content, for example: waiting for QR code login */
+    skipAssertData?: boolean;
+  } = {
+    skipAssertData: false,
+  },
+): void => {
+  if (resBody.errors) {
+    throw new Error(resBody.errors[0]?.message ?? 'Unknown error: ' + JSON.stringify(resBody.errors));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)